src/main/java/com/sendgrid/helpers/eventwebhook/EventWebhook.java

```diff
@@ -0,0 +1,54 @@
+package com.sendgrid.helpers.eventwebhook;
+
+import java.security.InvalidKeyException;
+import java.security.KeyFactory;
+import java.security.NoSuchAlgorithmException;
+import java.security.NoSuchProviderException;
+import java.security.Signature;
+import java.security.SignatureException;
+import java.security.interfaces.ECPublicKey;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.X509EncodedKeySpec;
+import java.util.Base64;
+
+/**
+ * This class allows you to easily use the Event Webhook feature. Read the docs
+ * for more details:
+ * https://sendgrid.com/docs/for-developers/tracking-events/event.
+ */
+public class EventWebhook {
+    /**
+     * Verify signed event webhook requests.
+     *
+     * @param publicKey: your verification key under Mail Settings
+     * @param payload:   event payload in the request body
+     * @param signature: value obtained from the
+     *                   'X-Twilio-Email-Event-Webhook-Signature' header
+     * @param timestamp: value obtained from the
+     *                   'X-Twilio-Email-Event-Webhook-Timestamp' header
+     */
+    public static boolean VerifySignature(String publicKey, String payload, String signature, String timestamp)
```

Comment:
Generally don't like statics unless there's a good reason. What else could this class be used for?

Reply:
it could be used for anything EventWebhook related

Reply:
Yeah, then I'd prefer it not be static so we could do some other stuff in here with some instance data.